### ISSUE TITLE
Make install error

### ISSUE THREAD

**Issue by Stikus** [biod/sambamba#459]
Hello, I'm trying to utilize `make install` part of `Makefile` and get this error.

```
install -m 0755 bin/sambamba /bin
install: cannot stat 'bin/sambamba': No such file or directory
make: *** [install] Error 1
Makefile:112: recipe for target 'install' failed
```
Looks like this [line](https://github.com/biod/sambamba/blob/master/Makefile#L112) is causing problem: 
```
install:
	install -m 0755 bin/sambamba $(prefix)/bin
```
According to code and my observations your binary is [called](https://github.com/biod/sambamba/blob/master/Makefile#L39):
```
OUT         = bin/sambamba-$(shell cat VERSION)
```
Maybe `bin/sambamba` should be modified to `$(OUT)`?
```
install:                                    
        install -m 0755 $(OUT) $(prefix)/bin
```
With this change everything is working for me as I see.

**Comment by pjotrp** [comment]
Thanks, will fix!

**Comment by pjotrp** [comment]
Fixed with 09ffba0822b2636d7d9882d954884cee8c1480c3
